serverless.ts

```diff
@@ -188,6 +198,19 @@ const serverlessConfiguration: AWS = {
           },
         }
       },
+      VacationRequests: {
+        Type: "AWS::DynamoDB::Table",
+        DeletionPolicy: "Delete",
+        Properties: {
+          TableName: "VacationRequests",
+          AttributeDefinitions: [{ AttributeName: "id", AttributeType: "S" }],
+          KeySchema: [{ AttributeName: "id", KeyType: "HASH" }],
+          ProvisionedThroughput: {
+            ReadCapacityUnits: 1,
+            WriteCapacityUnits: 1
+          },
+        }
+      },
       Software: {
```

Comment:
Remember to ad IAM role for you new table. Like with Software Registery or Questionnaires